src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDistribucion.kt

```diff
@@ -5,26 +5,17 @@ class CentroDistribucion(val ciudad:Ciudad){
     val vendedores = mutableListOf<Vendedor>()
 
     fun agregarVendedor(vendedor:Vendedor){
-        if(!vendedores.contains(vendedor)){
-            vendedores.add(vendedor)
-        }else{
+        if(vendedores.contains(vendedor)) {
             throw Exception("No se puede agregar dos veces el mismo vendedor")
         }
+        vendedores.add(vendedor)
     }
 
-    fun vendedorEstrella() : Vendedor? {
-        return vendedores.maxBy { it.puntajeCertificaciones() }
-    }
+    fun vendedorEstrella() : Vendedor? = vendedores.maxBy { it.puntajeCertificaciones() }
 
-    fun puedeCubrirCiudad(ciudad:Ciudad) : Boolean{
-        return vendedores.any { it.puedeTrabajarEn(ciudad) }
-    }
+    fun puedeCubrirCiudad(ciudad:Ciudad) : Boolean = vendedores.any { it.puedeTrabajarEn(ciudad) }
```

Comment:
Dato de color, si usás la sintaxis del `=`, no hace falta especificar el tipo que devuelve, porque lo toma de la expresión directamente.

```suggestion
    fun puedeCubrirCiudad(ciudad:Ciudad) = vendedores.any { it.puedeTrabajarEn(ciudad) }
```